### PR TITLE
New package: osm-gps-map-1.1.0

### PR DESCRIPTION
**Pull request by mustaqimM**

### CODE REVIEW

File: srcpkgs/osm-gps-map/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+maintainer="mustaqim <mustaqim+void@pm.me>"
+license="GPL-2.0-or-later"
+homepage="https://nzjrs.github.io/osm-gps-map"
+changelog="https://raw.githubusercontent.com/nzjrs/osm-gps-map/master/NEWS"
+distfiles="https://github.com/nzjrs/${pkgname}/archive/${version}.tar.gz"
+checksum=027004459f2f29232d4ca0c914f835e6c06b720aef1daa9b470d9d5e1b71159b
+
+pre_configure() {
+	./autogen.sh
+	: ${configure_script:=./configure}
```
Comment (maxice8) [inline srcpkgs/osm-gps-map/template]: remove these except autogen.sh

**Comment by mustaqimM** [comment]
@maxice8 Is it failing on Travis because `noarch=yes' isn't included?

File: srcpkgs/osm-gps-map/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'osm-gps-map'
+pkgname=osm-gps-map
+version=1.1.0
+revision=1
+build_style=configure
```
Comment (maxice8) [inline srcpkgs/osm-gps-map/template]: it is gnu-configure

File: srcpkgs/osm-gps-map/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+configure_args="--prefix=/usr --enable-gtk-doc"
```
Comment (maxice8) [inline srcpkgs/osm-gps-map/template]: with gnu-configure (See above) --prefix isn't necessary.

**Comment by maxice8** [comment]
> @maxice8 Is it failing on Travis because `noarch=yes' isn't included?

No, please apply the changes proposed.

File: srcpkgs/osm-gps-map/template [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'osm-gps-map'
+pkgname=osm-gps-map
+version=1.1.0
+revision=1
+build_style=gnu-configure
+configure_args="--enable-gtk-doc"
+hostmakedepends="gtk+3-devel libsoup-devel libtool gnome-common gtk-doc gobject-introspection pkg-config"
```
Comment (maxice8) [inline srcpkgs/osm-gps-map/template]: gtk+3-devel libsoup-devel should be in makedepends=

See the manual for the difference between checkdepends, depends, makedepends and hostmakedepends

**Comment by Chocimier** [comment]
Isn't that https://github.com/void-linux/void-packages/blob/master/srcpkgs/libosmgpsmap/template ?

**Comment by maxice8** [comment]
On Mon, Feb 11, 2019 at 04:47:15AM -0800, Piotr wrote:
> Isn't that https://github.com/void-linux/void-packages/blob/master/srcpkgs/libosmgpsmap/template ?
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/8506#issuecomment-462315527

yes, didn't notice that with the non upstream-following name

**Comment by maxice8** [comment]
This package is already in repo under the name libosmgpsmap.

Thanks @Chocimier for pointing it out

**Pull request closed without merging**